misago/users/tests/test_user_create_api.py

```diff
@@ -39,7 +39,7 @@ def test_empty_request(self):
     def test_invalid_data(self):
         """invalid request data errors with code 400"""
         response = self.client.post(
-            self.api_link, "false", content_type="application/json"
+            self.api_link, {}, content_type="application/json"
```

Comment:
This test was explicitly checking if non-dict data is handled by rest framework.

Reply:
venv/misago/lib/python3.6/site-packages/django/forms/widgets.py:258:
>       return data.get(name)
E       AttributeError: 'bool' object has no attribute 'get'